facebook/docusaurus
# refactor(utils): use Remark to parse Markdown server-side

## Motivation

- #hacktoberfest
- Fix #5659

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

I did the following to verify my changes work:
- add expectations and verified the snapshot change in `transforms absolute links in versioned docs` test (`linkify.test.ts`) to confirm md file links contained in html comments are not in the expected result (since comments are stripped out before looking for md broken links)
- run `npm run start:website:watch` locally and add the following lines in `website/docs/installation.md` file : 

    ```
    

    
    ```
    
    I saved the file and verified the warning is not shown anymore

## Related PRs

N/A
